.github/pull_request_template.md

```diff
@@ -2,4 +2,4 @@ PMM-0
 
 Build: SUBMODULES-0
 
-- [ ] Links to other linked pull requests (optional).
\ No newline at end of file
+- Links to related pull requests (optional).
```

Comment:
let's keep it as is.

Reply:
Well, it shows a task that normally serve as "todo" items, while in reality there is no action to take. Moreover, we seem never tick off those checkboxes. 

I'm attaching screenshots for better clarity.

<img width="1252" alt="Screenshot 2023-02-15 at 12 11 44" src="https://user-images.githubusercontent.com/81549/218984526-b8279fe0-1470-44ff-93be-4c68e49ab4a2.png">

<img width="864" alt="Screenshot 2023-02-15 at 12 14 44" src="https://user-images.githubusercontent.com/81549/218984767-68f0b4a4-6b0f-4157-8bf0-1c9cd4d25c52.png">

Question is: should the link to a PR be an action item?

On the contrary, there is a good example where checkboxes are totally legit and very useful - https://github.com/percona/pmm/issues/1541.

Reply:
Originally it was made as a checkbox to help tracking of merge order (i.e. make it visible what is merged and what is not). But GH renders icon that reflects PR status in front on the link and almost nobody uses these check boxes. So probably it make sense to remove them.